code/src/Components/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+import PostInput from './PostInput';
+import PostList from './PostList';
+import { MESSAGE_URL } from '../Urls';
+import Loader from './Loader';
+
+import './Style.css';
+
+export const App = () => {
+	const [messages, setMessages] = useState([]);
+	const [isLoading, setLoading] = useState(true);
+	useEffect(() => {
+		fetchMessages();
+		setInterval(fetchMessages, 5000);
+	}, []);
+
+	const fetchMessages = () => {
+		fetch(MESSAGE_URL)
+			.then(res => res.json())
+			.then(data => {
+				const filteredData = data.filter(post => post.message);
+				setMessages(filteredData);
+				setLoading(false);
+			})
+			.catch(error => console.error(error));
```

Comment:
Nice with the `.catch` here! I'll need to learn more about that 😄

Reply:
Thank you =)